Content.Shared/Clothing/SharedMagbootsComponent.cs

```diff
@@ -3,31 +3,24 @@
 using Robust.Shared.GameStates;
 using Robust.Shared.Serialization;
 
-namespace Content.Shared.Clothing
+namespace Content.Shared.Clothing;
+
+[NetworkedComponent()]
+public abstract class SharedMagbootsComponent : Component
 {
-    [NetworkedComponent()]
-    public abstract class SharedMagbootsComponent : Component
-    {
-        [DataField("toggleAction", required: true)]
-        public InstantAction ToggleAction = new();
+    [DataField("toggleAction", required: true)]
+    public InstantAction ToggleAction = new();
 
-        public abstract bool On { get; set; }
+    public abstract bool On { get; set; }
```

Comment:
The On property doesn't need to be abstract so you can put the entire component in shared.